client/html/public/scripts/login.js

```diff
@@ -10,31 +10,49 @@ function initialize() {
 
 // Renders the user dropdown menu
 function displayUsers() {
-  let [currentUser] = userList;
+  let [currentAccount] = accounts;
 
-  document.cookie.split(';').forEach((c) => {
-    const [cookieName, cookieValue] = c.split('=');
+  document.cookie.split('; ').forEach((cookie) => {
+    const [cookieName, cookieValue] = cookie.split('=');
     if (cookieName === 'user') {
-      currentUser = cookieValue
+      currentAccount = cookieValue;
     }
   })
 
   const menu = document.getElementById('login-menu');
+  const tenantUsers = accounts.reduce((tenantUsers, account) => {
+    const [tenant, user] = account.split('/').map((account) => account.trim());
+    tenantUsers[tenant] ??= [];
+    tenantUsers[tenant].push({account, name: user});
+    return tenantUsers;
+  }, {});
+
+  const [currentTenant, currentUser] = currentAccount.split('/').map((account) => account.trim());
+
   menu.innerHTML = `\
   <form id="login">
-    <label for="user-picker">Current tenant / user</label>
-    <select id="user-picker" name="user" class="user" onchange="login(true)">
-      ${userList.map((user) => 
-        `<option ${user == currentUser ? "selected" : ""}>${user}</option>`
+    <label for="account-select">User</label>
+    <select id="account-select" name="account" class="account" onchange="login(true)">
```

Comment:
On Firefox, the drop-down is squished all the way to the right-hand side of the page. Might not be much we can do about it, but it looks less good than on Chrome, where there is a nice gap of air.

(Don't know how to make a screen grab, as the drop-down disappears whenever I start the `Screenshot` app 😅 )

Reply:
looks fine on firefox?
![Screen Shot 2022-12-02 at 11 45 01 AM](https://user-images.githubusercontent.com/5966087/205373980-68c8c1a4-22b4-4545-937c-d55be3370d0e.png)